src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3638,7 +3638,10 @@ bool os::pd_release_memory(char* addr, size_t bytes) {
     // Handle mapping error. We assert in debug, unconditionally print a warning in release.
     if (err != nullptr) {
       log_warning(os)("bad release: [" PTR_FORMAT "-" PTR_FORMAT "): %s", p2i(start), p2i(end), err);
+#ifdef ASSERT
+      os::print_memory_mappings((char*)start, bytes, tty);
```

Comment:
I think this will cause lock rank conflicts if you leave it in.  Please see [here](https://github.com/openjdk/jdk/pull/22745#discussion_r1890345578) and [here](https://github.com/openjdk/jdk/pull/22745#discussion_r1894197615) for more context.